user_threshold/models/res_users.py

```diff
@@ -20,32 +20,24 @@ class ResUsers(models.Model):
         'Exempt User From User Count Thresholds',
     )
 
-    def __init__(self, pool, cr):
+    @api.model_cr
+    def _register_hook(self, pool, cr):
```

Comment:
Why not simply `def _register_hook(self):`?

Reply:
Annnnnnd now I have actually RTFM. This is why blindly copying code is probably a bad idea lmao

Reply:
You should have copy-pasted my patch :D